### ISSUE TITLE
Option to disable all traces at once

### ISSUE THREAD

**Issue by bblanchon** [bblanchon/ArduinoTrace#2]

**Comment by DarioGHub** [comment]
Thank you for your work on this utility. If a sketch is written thusly,
```
#define ARDUINOTRACE_ENABLE 1                 // reminder that we now have a new option
#include <ArduinoTrace.h>
```

the IDE gives `warning: "ARDUINOTRACE_ENABLE" redefined`.
and all traces are now disabled.

If the intention was to enable traces if that option were previously undefined, maybe remove these 2 lines 
```
#ifndef ARDUINOTRACE_ENABLE
#define ARDUINOTRACE_ENABLE 1
//#else
//#define ARDUINOTRACE_ENABLE 0
#endif
```

**Comment by bblanchon** [comment]
Hi @DarioGHub,

Thank you for reporting this mistake; I fixed it in v1.1.1.

Regards,
Benoit
